rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -215,8 +225,9 @@ rmw_create_client(const rmw_node_t * node,
         _unregister_type(participant, info->response_type_support_, info->typesupport_identifier_);
       }
     } else {
-      fprintf(stderr,
-        "[rmw_fastrtps] leaking type support objects because node impl is null\n");
+      RCUTILS_LOG_ERROR_NAMED(
+        "rmw_fastrps_cpp",
```

Comment:
fixed in https://github.com/ros2/rmw_fastrtps/pull/138/commits/cda8cb8d82e9b02173cfc7e2fa80f71ce20a18a0